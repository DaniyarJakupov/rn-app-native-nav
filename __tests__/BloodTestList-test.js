import 'react-native';
import React from 'react';
import BloodTestList from '../app/screens/BloodTestList';
import renderer from 'react-test-renderer';

jest.useFakeTimers();

it('BloodTestList snapshot', () => {
  const tree = renderer.create(<BloodTestList />).toJSON;
  expect(tree).toMatchSnapshot();
});
