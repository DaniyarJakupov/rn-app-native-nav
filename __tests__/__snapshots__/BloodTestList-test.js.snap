// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BloodTestList snapshot 1`] = `[Function]`;
